lib/hanami/generators/application/app/Gemfile.tt

```diff
@@ -15,7 +15,7 @@ gem 'hanami-mailer',      require: false, github: 'hanami/mailer'
 gem 'hanami-assets',      require: false, github: 'hanami/assets'
 gem 'hanami',                             github: 'hanami/hanami'
 <%- else -%>
-gem 'hanami',       '<%= Hanami::VERSION %>'
+gem 'hanami',       '~> <%= Hanami::VERSION_WITHOUT_TINY %>'
```

Comment:
@davydovanton We should unify with `hanami-model` definition and use `<%= config[:hanami_version] %>`.
The logic for defining that version should take care of `Hanami::VERSION`, but **NOT** to define a new constant.

Example:

```ruby
version = Hanami::VERSION.scan(/\A\d{1,2}\.\d{1,2}/).first
```